backend/game/cards/keep_cards/health_manipulation_cards/even_bigger.py

```diff
@@ -0,0 +1,17 @@
+from game.cards.card import Card
+from game.cards.keep_card import KeepCard
+
+
+class EvenBigger(KeepCard):
+    def __init__(self):
+        Card.__init__(self, "Even Bigger", 4,
+                      "+2[health] when you buy this card.maximum [health] increased to 12 as long as you own this card",
```

Comment:
This card will need to have the effect of losing the maximum health set back to the original value but this could either be done now or in a later iteration